ppr-ui/.eslintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true
+  },
+  extends: [
+    'plugin:vue/essential',
+    '@vue/standard',
+    '@vue/typescript'
+  ],
+  rules: {
+    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    'max-len': ['warn', { code: 120, ignoreRegExpLiterals: true }]
```

Comment:
Can we add
'no-unused-vars': process.env.NODE_ENV === 'production' ? 'error' : 'warn',